.github/workflows/phpunit-summary-when-unrelated.yml

```diff
@@ -0,0 +1,68 @@
+# This workflow is provided via the organization template repository
+#
+# https://github.com/nextcloud/.github
+# https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization
+
+name: PHPUnit
+
+on:
+  pull_request:
+    paths-ignore:
+      - '.github/workflows/**'
+      - 'appinfo/**'
+      - 'lib/**'
+      - 'templates/**'
+      - 'tests/**'
+      - 'vendor/**'
+      - 'vendor-bin/**'
+      - '.php-cs-fixer.dist.php'
+      - 'composer.json'
+      - 'composer.lock'
+
+permissions:
+  contents: read
+
+jobs:
+  summary-mysql:
+    permissions:
+      contents: none
+    runs-on: ubuntu-latest
+
+    name: phpunit-mysql-summary
+
+    steps:
+      - name: Summary status
+        run: 'echo "No PHP files changed, skipped PHPUnit"'
```

Comment:
Is this step only to print this? That the expected result with this GitHub Action?

Reply:
Ref is https://github.com/nextcloud/.github/pull/97
It is needed so we can require the PHPunit tests again.
But don't want to make a new PR which otherwise causes a lot of load on CI